docs/source/en/model_doc/llama2.md

```diff
@@ -55,6 +55,31 @@ come in several checkpoints they each contain a part of each weight of the model
 
 This model was contributed by [Arthur Zucker](https://huggingface.co/ArthurZ) with contributions from [Lysandre Debut](https://huggingface.co/lysandre). The code of the implementation in Hugging Face is based on GPT-NeoX [here](https://github.com/EleutherAI/gpt-neox). The original code of the authors can be found [here](https://github.com/facebookresearch/llama).
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with LLaMA2. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+- A blog on how to [understand Llama 2's features and its collaboration with Hugging Face](https://huggingface.co/blog/llama2).
+- A blog on [an introduction to Llama 2](https://www.philschmid.de/llama-2) by Phil Schmid.
```

Comment:
While nice to attribute credit, I don't think its necessary here since Philip is considered a "official Hugging Face" source.

```suggestion
- [LLaMA 2 - Every Resource you need](https://www.philschmid.de/llama-2), a compilation of relevant resources to learn about LLaMA 2 and how to get started quickly.
```